tajo-docs/src/main/sphinx/partitioning/column_partitioning.rst

```diff
@@ -11,22 +11,40 @@ How to Create a Column Partitioned Table
 You can create a partitioned table by using the ``PARTITION BY`` clause. For a column partitioned table, you should use
 the ``PARTITION BY COLUMN`` clause with partition keys.
 
-For example, assume there is a table ``orders`` composed of the following schema. ::
+For example, assume a table with the following schema.
 
-  id          INT,
-  item_name   TEXT,
-  price       FLOAT
+.. code-block:: sql
+
+  id     INT,
+  name   TEXT,
+  grade  TEXT
```

Comment:
Maybe you intend that this table has the five columns of id, name, grade, country, and city. If so, how about showing all columns as the below example of ```student_source```? It will be much easier to understand.